internal/ctrlutils/ingress-status.go

```diff
@@ -13,8 +14,9 @@ import (
 	"github.com/go-logr/logr"
 	"github.com/kong/deck/file"
 	apiv1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
1. package name [guidelines](https://go.dev/blog/package-names) (I assume they also apply to package name aliases):

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps

2. why alias this name at all? there's explicit guidance against this [link](https://github.com/golang/go/wiki/CodeReviewComments#imports)


suggestion: just leave the default `errors` unless there's a pkg name collision (I don't see one)

Reply:
This [adds stdlib `errors` also](https://github.com/Kong/kubernetes-ingress-controller/pull/2005/commits/53682e2dfba7c0a7d0b9e0a29d38f5ea7088ce39#diff-acd60b643e0d662a409aa00230a4a34bb0ae4dee18903d773d757277edc26340R5), is there some way that doesn't conflict without the alias? 69bfc022 to match the rest of the convention.

Reply:
Oh, of course it does conflict without the alias - I overlooked the `"errors"` at the top. My bad.